utils/pub/pubDates.ts

```diff
@@ -31,6 +32,15 @@ export const getPubPublishedDate = (pub: Pub, includeCustomPublishedAt = true) =
 	return null;
 };
 
+export const getPubPublishedDateString = (pub: Pub): string => {
```

Comment:
The logic here is subtly different than `getPubPublishedDate`: it calls `getLocalDateMatchingUtcCalendarDate` on either a Release date of a `customPublishedAt`, but `getPubPublishedDate` will only do the former.

I suggest calling `getPubPublishedDate(pub, true)` from this function, and also returning `null` instead of the empty string when there's no publication date available.